README.md

```diff
@@ -297,14 +311,19 @@ result type has to implement `Into<ContractError>`, where `ContractError` is a c
 error type - it will all be commonized in the generated dispatching function (so
 entry points have to return `ContractError` as its error variant).
 
-Messages equivalent to execution messages are generated. Let's create an entry point:
+Messages equivalent to execution messages are generated.
+Again entry point is already generated and ready to be used:
 
 ```rust
-use sylvia::cw_std::{Deps, Binary};
-
-#[entry_point]
-fn query(deps: Deps, env: Env, msg: ContractQueryMsg) -> Result<Binary, ContractError> {
-    msg.dispatch(&CONTRACT, (deps, env))
+pub mod entry_points {
+    #[entry_points]
```

Comment:
This is not valid.

I am not sure if it is a point to show entry points again and again, reading it I might be confused if I need to add this code or what.